spec/model/load_balancer_spec.rb

```diff
@@ -64,4 +64,60 @@
       expect(lb.load_balancers_vms.count).to eq(0)
     end
   end
+
+  describe "need_certificates?" do
```

Comment:
Commit message is: " ACME Cert integration tests ".... I think this is ambiguous because "integration test" has a conflicting meaning. I'm sure you can phrase-smith it to not make someone wonder if this contacts a live acme service, as "e2e" (which could be called an "integration test") does.

Reply:
Updated